x-pack/plugins/security_solution/server/lib/detection_engine/routes/signals/open_close_signals_route.ts

```diff
@@ -132,3 +112,57 @@ export const setSignalsStatusRoute = (
     }
   );
 };
+
+const updateSignalsStatusByIds = async (
+  status: SetSignalsStatusSchemaDecoded['status'],
+  signalsId: string[],
+  spaceId: string,
+  esClient: ElasticsearchClient
+) =>
+  esClient.bulk({
+    index: `${DEFAULT_ALERTS_INDEX}-${spaceId}`,
+    refresh: 'wait_for',
+    body: signalsId.flatMap((signalId) => [
+      {
+        update: { _id: signalId, _index: `${DEFAULT_ALERTS_INDEX}-${spaceId}` },
+      },
+      {
+        script: getUpdateSignalStatusScript(status),
+      },
+    ]),
+  });
+
+const updateSignalsStatusByQuery = async (
```

Comment:
Do we want to either deprecate or add a note about wanting to move away form this?